content/en/docs/concepts/architecture/master-node-communication.md

```diff
@@ -104,6 +104,17 @@ all traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the network in
 which the nodes are running.
 
-SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. A replacement for this communication channel is being designed.
+SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. APIServer network proxy is a replacement for this communication channel.
+
+### APIServer network proxy (alpha)
+
+As a replacement to the SSH tunnels, Kubernetes supports APIServer network proxy
+to protect the Master -> Cluster communication paths. The feature is still
+alpha. It supports deploying on GCE via the
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)
+script. Run `KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE=true
+./cluster/kube-up.sh` to start the cluster with APIServer network proxy on.
+You can find more details in the [apiserver-network-proxy
+repository](https://github.com/kubernetes-sigs/apiserver-network-proxy).
```

Comment:
This seems very specific to a particular cloud infrastructure provider. Instead, link to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ (search for links to `/docs/reference/command-line-tools-reference/feature-gates/` to see how other pages typically do this); readers can then work out what approach they need to enable the feature manually.

Reply:
Unfortunately this code is not behind a feature flag. It's is enabled/disabled by this environment variable in kube-up.sh.

Reply:
I think you need to explain how someone might set this up and use it, e.g. explain the apiserver settings, the control plane proxy binary, and the agent.